services/gateway/apis/http-version3/swagger/definitions/blocks.json

```diff
@@ -0,0 +1,199 @@
+{
+    "Block": {
+        "type": "object",
+        "required": [
+            "id",
+            "version",
+            "height",
+            "timestamp",
+            "generatorAddress",
+            "transactionRoot",
+            "assetsRoot",
+            "stateRoot",
+            "maxHeightPrevoted",
+            "maxHeightGenerated",
+            "validatorsHash",
+            "aggregateCommit",
+            "signature",
+            "isFinal"
+        ],
+        "properties": {
+            "id": {
+                "type": "string",
+                "format": "id",
+                "minLength": 1,
+                "maxLength": 64,
+                "example": "01967dba384998026fe028119bd099ecf073c05c045381500a93d1a7c7307e5b",
+                "description": "Unique identifier of the block.\nDerived from the block signature.\n"
+            },
+            "version": {
+                "type": "integer",
+                "example": 0,
+                "minimum": 0,
+                "description": "Versioning for future upgrades of the Lisk protocol."
+            },
+            "height": {
+                "type": "integer",
+                "example": 8344448,
+                "minimum": 1,
+                "description": "Height of the network, when the block got forged.\nThe height of the networks represents the number of blocks,\nthat have been forged on the network since Genesis Block.\n"
```

Comment:
```suggestion
                "description": "Height of the network, when the block got forged.\nThe height of the networks represents the number of blocks, that have been forged on the network since Genesis Block.\n"
```